documentation/asciidoc/topics/proc_defining_environment_var.adoc

```diff
@@ -9,8 +9,10 @@ ifdef::downstream[]
 NOTE: If you are using the {openshift} Console, you must first install the {ispn_operator} and then edit the existing subscription.
 endif::downstream[]
 
-Environment variables in your subscription automatically propagate to each {brandname} pod that is managed by the corresponding Operator.
-The `spec.config.env` field is an array of environment variable definitions.
+Define environment variables using the `spec.config.env` field.
+Set the name of the first variable as `ADDITIONAL_VARS`.
+This variable is a JSON array that includes the names of environment variables to be automatically propagated to each {brandname} Server pod managed by the associated Operator.
+
 When one environment variable references another, you must define the variables in a specific order for proper referencing and processing.
```

Comment:
Do we need this sentence here given it's general rule for any OpenShift resource (cc @ryanemerson )? It's not related to propagating env vars to the server pods

Reply:
That's true. Let's keep it focused and remove this.